src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoServlet.scala

```diff
@@ -50,7 +51,7 @@ class EasyAuthInfoServlet(app: EasyAuthInfoApp) extends ScalatraServlet
   }
 
   private def getUUID: Try[UUID] = {
-    Try { UUID.fromString(params("uuid")) }
+    params("uuid").toUUID.toTry
```

Comment:
doesn't this require a change in the `respond` pattern match below as well?

Reply:
I don't think so.  We come into `respond` only when UUID is valid. If UUID is invalid, we get `BadRequest(t.getMessage)`.